Sources/FndBase/Configs/LegacyConfig.swift

```diff
@@ -0,0 +1,213 @@
+            _torusNodeTSSEndpoints: try getTSSEndpoints(network: .legacy(network)),
+            _torusIndexes: [1, 2, 3, 4, 5],
+            _torusNodePub: [
+//                .init(_X: "1363aad8868cacd7f8946c590325cd463106fb3731f08811ab4302d2deae35c3", _Y: "d77eebe5cdf466b475ec892d5b4cffbe0c1670525debbd97eee6dae2f87a7cbe"),
+//                .init(_X: "7c8cc521c48690f016bea593f67f88ad24f447dd6c31bbab541e59e207bf029d", _Y: "b359f0a82608db2e06b953b36d0c9a473a00458117ca32a5b0f4563a7d539636"),
+//                .init(_X: "8a86543ca17df5687719e2549caa024cf17fe0361e119e741eaee668f8dd0a6f", _Y: "9cdb254ff915a76950d6d13d78ef054d5d0dc34e2908c00bb009a6e4da701891"),
+//                .init(_X: "25a98d9ae006aed1d77e81d58be8f67193d13d01a9888e2923841894f4b0bf9c", _Y: "f63d40df480dacf68922004ed36dbab9e2969181b047730a5ce0797fb6958249"),
+//                .init(_X: "d908f41f8e06324a8a7abcf702adb6a273ce3ae63d86a3d22723e1bbf1438c9a", _Y: "f977530b3ec0e525438c72d1e768380cbc5fb3b38a760ee925053b2e169428ce"),
+                .init(_X: "f74389b0a4c8d10d2a687ae575f69b20f412d41ab7f1fe6b358aa14871327247", _Y: "54e3a73098ed9bced3ef8821736e9794f9264a1420c0c7ad15d2fa617ba35ef7"),
+                .init(_X: "bc38813a6873e526087918507c78fc3a61624670ee851ecfb4f3bef55d027b5a", _Y: "ac4b21229f662a0aefdfdac21cf17c3261a392c74a8790db218b34e3e4c1d56a"),
+                .init(_X: "b56541684ea5fa40c8337b7688d502f0e9e092098962ad344c34e94f06d293fb", _Y: "759a998cef79d389082f9a75061a29190eec0cac99b8c25ddcf6b58569dad55c"),
+                .init(_X: "7bcb058d4c6ffc6ba4bfdfd93d141af35a66338a62c7c27cdad2ae3f8289b767", _Y: "336ab1935e41ed4719e162587f0ab55518db4207a1eb36cc72303f1b86689d2b"),
+                .init(_X: "bf12a136ef94399ea098f926f04e26a4ec4ac70f69cce274e8893704c4951773", _Y: "bdd44828020f52ce510e026338216ada184a6867eb4e19fb4c2d495d4a7e15e4"),
+                
+            ]
+        )
+    case .AQUA :
+        return .init(
+            _currentEpoch: "1",
+            _torusNodeEndpoints: [
+                "https://node-1.cluster-2.aqua.web3auth.io/jrpc",
+                "https://node-2.cluster-2.aqua.web3auth.io/jrpc",
+                "https://node-3.cluster-2.aqua.web3auth.io/jrpc",
+                "https://node-4.cluster-2.aqua.web3auth.io/jrpc",
+                "https://node-5.cluster-2.aqua.web3auth.io/jrpc",
+            ],
+            _torusNodeSSSEndpoints: try getSSSEndpoints(network: .legacy(network)),
+            _torusNodeRSSEndpoints: try getRSSEndpoints(network: .legacy(network)),
+            _torusNodeTSSEndpoints: try getTSSEndpoints(network: .legacy(network)),
+            _torusIndexes: [1, 2, 3, 4, 5],
+            _torusNodePub: [
+                .init(_X: "7d376a4a7ce6bba7a8a49001aaa2de3f24e0f56ee73a77c6678ec00363616f25", _Y:
+                    "d5752873e522f98b4ded27690aadfda57ff5dc2fd0c3055b9d83663441667010"),
+
+                .init(_X: "0e1b5debe1d9b42eb9917a5fe84fa5f48cae7a770b1f49f0546fbe26b1efaff3", _Y:
```

Comment:
No need of all these static keys for aqua, cyan and celeste we never use them anywhere, we always fetch from smart contract for multi cluster networks